src/selector.rs

```diff
@@ -37,6 +38,7 @@ impl PseudoClassElement {
     fn weight(&self) -> u32 {
         match self {
             PseudoClassElement::Hover => 10,
+            PseudoClassElement::Active => 11,
```

Comment:
Both `Hover` and `Active` should have weight 10, since, `Specifity` algorithm works with three weights. 1-1-1, in this case, pseudo-classes have weight of 0-1-0, so 10 (the same as `Hover`)

Reply:
Totally forgot, that I wanted to check this. Is now updated.